services/verified.ts

```diff
@@ -0,0 +1,49 @@
+import { FPLBOT_API_BASEURL } from "../utils/envconfig";
+
+export interface VerifiedEntry {
+  entryId: number;
+  slug: string;
+  teamName: string;
+  realName: string;
+  plName: string;
+  playsForTeam: string;
+  shirtImageUrl: string;
+  imageUrl: string;
+  pointsThisGw: number;
+  totalPoints: number;
+  overallRank: number;
+  captain: string;
+  viceCaptain: string;
+  chipUsed?: ChipType;
+  movement: number;
+  selfOwnershipWeekCount: number;
+  selfOwnershipTotalPoints: number;
+  gameweek: number;
+}
+
+export type ChipType = 
+    | "3xc"
+    | "wildcard"
+    | "freehit"
+    | "bboost";
+
+export function getVerifiedEntries(): Promise<VerifiedEntry[]> {
+    return fetch(`${FPLBOT_API_BASEURL}/fpl/verified`)
```

Comment:
Blir veldig glad hvis du bruker samme pattern som meg på http-ting. Litt fordi jeg er religiøst overbevist om at det er den beste måten™️ å gjøre det på, men mest for å gjøre det konsekvent i repoet 😄 Da tvinges du riktignok til å legge til feilhåndtering på de nye sidene dine også